src/habluetooth/base_scanner.py

```diff
@@ -571,8 +570,6 @@ def _async_on_advertisement_internal(
             # https://github.com/hbldh/bleak/blob/222618b7747f0467dbb32bd3679f8cfaa19b1668/bleak/backends/scanner.py#L203
             #
             # _rssi is deprecated, will be removed in newer bleak
```

Comment:
```suggestion
            #
            # _rssi is deprecated, will be removed in newer bleak
```
These can be removed now.